### PR TITLE
PAR-1698: Add coachmark

### PR DESCRIPTION
**Pull request by fredrikloberg**
New coachmark component to introduce new features.

Screenshot 2023-02-24 at 11 10 06
Screenshot 2023-02-24 at 11 10 14

### CODE REVIEW

**Comment by atraastrum** [comment]
Looks like everything is in alphabetical order except `Coachmark`. Should `Coachmark` be after `Checkbox` under **Components**?

File: src/Nordea/Components/Coachmark.elm [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+import Nordea.Resources.I18N exposing (Translation)
+import Nordea.Resources.Icons as Icons
+import Nordea.Themes as Themes
+import Set
+
+
+type OptionalConfig
+    = HighlightedClass String
+    | Placement Tooltip.Placement
+    | ShowPage (Maybe Int)
```
Comment (vetlesolgaard) [inline src/Nordea/Components/Coachmark.elm:66]: Does this depict the step number of the "coachmark tutorial"? I associate `page` with a full size page. Could `step` be a bit more accurate? 🤔

File: src/Nordea/Resources/Icons.elm [hunk 2]
```diff
@@ -881,6 +882,22 @@ outerLink attrs =
         ]
 
 
+lightBulb : List (Attribute msg) -> Html msg
+lightBulb attrs =
```
Comment (vetlesolgaard) [inline src/Nordea/Resources/Icons.elm:886]: Could we just slide this into the `Icons` story page as well? 😁

File: src/Nordea/Components/Coachmark.elm (hunk 1 above)
Comment (fredrikloberg) [reply src/Nordea/Components/Coachmark.elm]: Agreed, changed it to step

**Review by vetlesolgaard — APPROVED** [review]
Nice work!🙌

**Pull request merged**